pootle/templates/core/_top_scorers_as_ul.html

```diff
@@ -0,0 +1,22 @@
+{% load i18n humanize profile_tags %}
+{% if top_scorers %}
+<label>{% trans 'Top contributors:' %}</label>
+<ul class="top-scorers">
+  {% for user in top_scorers %}
+    <li class="top-scorer">
+      <div class="place">{% blocktrans with position=forloop.counter %}#{{ position }}{% endblocktrans %}</div>
+      <div class="user">
+        <a href="{{ user.get_absolute_url }}">
+          <img src="{{ user|gravatar:20 }}" class="avatar" />
+          <span class="user-name">{{ user.display_name }}</span>
+        </a>
+      </div>
+      <div class="number">{% if user.public_total_score > 0 %}+{% endif %}{{ user.public_total_score|floatformat:0|intcomma }}</div>
+    </li>
+  {% endfor %}
+</ul>
+{% else %}
+<div class="top-scorers no-results">
+  {% trans 'No user contributions for last 30 days.' %}
```

Comment:
@julen I reckon that we'll have to implement some caching system (similar to what we use for stats). And I'm afraid that it will be hard to stay with this generic approach.
But if you have thoughts how to avoid ad-hoc things atm I'd be happy to address them.